### PR TITLE
Added missing login info to initial setup instructions

### PR DESCRIPTION
**Pull request by acorncom**
If you haven't used Divshot before, you need to authorize your app ...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,14 @@ From within your Ember CLI application run:
 ember generate divshot
 ```
 
+### Login to Divshot
+
+From within your Ember CLI application run:
+
+```bash
+ember login divshot
```
Comment (csantero) [inline README.md]: Shouldn't this be `ember divshot login`?

Comment (rwjblue) [inline README.md]: Yes, I believe it should be.

**Comment by elwayman02** [comment]
This PR should probably also add `ember divshot init` to create a new divshot app

**Comment by acorncom** [comment]
Sorry about the delay on this ...

**Pull request closed without merging**